### PR TITLE
Adds `name` option to groups

### PR DESCRIPTION
**Pull request by richsilv**
Sometimes it's useful to know which group the current route belongs to (for example, is it only for logged-in users?).  Currently, that's not possible unless you're using prefixes for that group.

Allowing the user to pass a `name` option resolves this as it allows you to use `FlowRouter.current().route.group.name` to identify the current group.

### CODE REVIEW

File: package.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Package.describe({
   name: 'kadira:flow-router',
   summary: 'Carefully Designed Client Side Router for Meteor',
-  version: '2.3.0',
+  version: '2.3.0_1',
```
Comment (arunoda) [inline package.js]: Don't increment this.

Comment (richsilv) [inline package.js]: Apologies, that was just so I could tell I was using the new version locally.

**Comment by arunoda** [comment]
One more thing?
Is it possible to have a test case for this?

**Comment by richsilv** [comment]
:+1:

**Comment by arunoda** [comment]
Awesome.

**Comment by serkandurusoy** [comment]
I have two quick question:

1) What happens if groups are nested? Which name would the method return?
2) Does it not make sense to create getGroupName() that depends on _routeClosed so that we can get the name reactively?

**Comment by richsilv** [comment]
@serkandurusoy 
1. `name` is just a property rather than a method, so you can get the deepest-nested group name with `FlowRouter.current().route.group.name`, and its parent group's name with `FlowRouter.current().route.group.parent.name`, and so on.
2. If you want to do something reactive with name, you just need to put `FlowRouter.watchPathChange()` in your reactive function, and the computation will rerun whenever the current group name changes (or its parent group, etc.).

**Comment by serkandurusoy** [comment]
Thanks this has been really helpful, especially with an "active route group" helper I can now use in a deeply nested menu and route structure. :clap:

**Comment by arunoda** [comment]
Btw: Shall we add this into the README?
Could you send me a PR to the group section with how to access this.

**Pull request merged**